cpp/sopt/objective_functions.h

```diff
@@ -31,10 +30,8 @@ std::function<t_real(T)> const unconstrained_l1_regularisation(
     const t_real &gamma, const t_real &sig, const T &y,
     const sopt::LinearTransform<T> &measurement_operator,
     const sopt::LinearTransform<T> &wavelet_operator);
-}  // namespace objective_functions
-}  // namespace sopt
-namespace sopt {
-namespace objective_functions {
+} // namespace sopt::objective_functions
+namespace sopt::objective_functions {
 
```

Comment:
Why does this close the `sopt:objective_functions` namespace and then immediately repoen it? Would it be better to just remove lines 34-37 so the namespace continues to line 70?

Reply:
I found this exact thing in one or two files. But I thought this PR should solely be about a 1:1 replacement of the current codebase.

I agree that closing and reopening a namespace immediately isn't particularly useful, and I am happy to roll up this and other leftover code-quality issues into a smaller, separate PR.

Reply:
Yes I agree that having this one just be a 1:1 switch and make other changes separately sounds good.

Reply:
Recorded in #390 and should be fixed by #392.